Tests/Tests/SDAnimatedImageTest.m

```diff
@@ -483,28 +483,39 @@ - (void)test28AnimatedImageAutoPlayAnimatedImage {
     SDAnimatedImage *image = [SDAnimatedImage imageWithData:[self testAPNGPData]];
     imageView.image = image;
 
-    expect(imageView.animating).equal(0);
+    #if SD_UIKIT
+        expect(imageView.animating).equal(0);
+    #else
+        expect(imageView.animates).equal(0);
+    #endif
     
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.5 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
-        expect(imageView.animating).equal(0);
+        #if SD_UIKIT
+            expect(imageView.animating).equal(0);
+        #else
+            expect(imageView.animates).equal(0);
+        #endif
         
         #if SD_UIKIT
             [imageView startAnimating];
         #else
             imageView.animates = YES;
         #endif
-        
-        expect(imageView.animating).equal(1);
     });
     
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
-        expect(imageView.animating).equal(1);
-
+        #if SD_UIKIT
+            expect(imageView.animating).equal(1);
```

Comment:
😂...Can you change this into the `.equal(YES)` ? Above change into `.equal(NO)`. The 1 and 0 does not always means YES or NO because it's a signed char type.